powerhome/playbook
# [PLAY-302] -  Button Kit Icon Props + Fixing Examples

#### Screens

#### Button with icons, now displaying on left or right in both react and rails version:
![Screen Shot 2022-09-12 at 4 20 59 PM](https://user-images.githubusercontent.com/73710701/189751238-7942064b-99ee-4a25-97fd-1420385aec5b.png)

#### Button with block content actually displaying block content in react and rails:
![Screen Shot 2022-09-12 at 2 00 48 PM](https://user-images.githubusercontent.com/73710701/189724270-586bb924-dcab-4334-8793-1d1efe370705.png)

#### Breaking Changes

No breaking changes. Added props to display icon on left or right of text in 'button icon options'. Also fixed 'Button with block content' example to actually display block content.

#### Runway Ticket URL

[Runway Story](https://nitro.powerhrg.com/runway/backlog_items/PLAY-302)

#### How to test this

Test in milano env. Navigate to Button kit page and make sure 'button icon options' and 'button with block content' are displaying as explained.

#### Checklist:
